### PR TITLE
Added Cobalt test cases

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Following are the test cases that are added in this PR : 
Cobalt_State_001.test.js 
Cobalt_State_002.test.js 
Cobalt_URL_001.test.js 
Cobalt_URL_002.test.js 
Cobalt_URL_003.test.js 
Cobalt_Visibility_001.test.js 
Cobalt_Visibility_002.test.js 
Cobalt_Visibility_003.test.js

### CODE REVIEW

File: src/tests/youtube/Cobalt_URL_001.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      description: 'Activate Cobalt browser',
+      test: pluginActivate,
+      params: constants.youTubePlugin,
+      assert: 'resumed',
+    },
+    {
+      description: 'Get the default URL Loaded on Cobalt',
+      sleep: 10,
+      test() {
+        return getCobaltUrl.call(this)
```
Comment (wouterlucas) [inline src/tests/youtube/Cobalt_URL_001.test.js:27]: Does this test actually work? I'm not sure if the URL functionality is available on Cobalt?

File: src/tests/youtube/Cobalt_URL_002.test.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      description: 'Activate Cobalt plugin',
+      test: pluginActivate,
+      params: constants.youTubePlugin,
+      assert: 'resumed',
+    },
+    {
+      description: 'Load URL and check the response',
+      sleep: 10,
+      test() {
+        return setCobaltUrl.call(this, this.$context.read('url'))
```
Comment (wouterlucas) [inline src/tests/youtube/Cobalt_URL_002.test.js:32]: Same here, suprised this is actually available. Does Cobalt really navigate to another URL?

File: src/tests/youtube/Cobalt_URL_001.test.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/tests/youtube/Cobalt_URL_001.test.js:27]: Yeah Wouter. This is working and it shows 'https://www.youtube.com/tv'.

File: src/tests/youtube/Cobalt_URL_002.test.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/tests/youtube/Cobalt_URL_002.test.js:32]: Yeah Wouter. This is working and it is navigating to google URL

**Review by wouterlucas — APPROVED** [review]
LGTM!

**Pull request merged**